indico/modules/logs/controllers.py

```diff
@@ -52,6 +57,21 @@ def _process(self):
                                               logs_api_url=url_for('.api_category_logs', self.category))
 
 
+class RHUserLogs(RHUserBase):
+    """Show the modification/action log for the user."""
+
+    def _check_access(self):
+        if not session.user.is_admin:
+            raise Forbidden
```

Comment:
To be consistent with other admin-only user endpoints:

```suggestion
        RHUserBase._check_access(self)
        if not session.user.is_admin:
            raise Forbidden
```